app/models/discussion.rb

```diff
@@ -9,10 +9,14 @@ class Discussion < ApplicationRecord
 
   # Instance methods
 
+  def notifiable?
+    resource&.user&.email&.present? ? true : false
```

Comment:
Remove the last safe-navigation operator to avoid the ternary expression:

```diff
-resource&.user&.email&.present? ? true : false 
+resource&.user&.email.present?
```